docs/user/notifications/pagerduty.rst

```diff
@@ -0,0 +1,26 @@
+Pagerduty
+---------
+
+Notify `Pagerduty <https://www.pagerduty.com/>`_ of a new alert status. If the alert ended then notification will ``resolve`` pagerduty incident.
```

Comment:
this sounds a bit weird "if the alert ended then notification will ``resolve`` pagerduty incident."